INSTALL

```diff
@@ -9,12 +10,12 @@ Please see https://github.com/anura-engine/anura/wiki/Building-On-Windows for Wi
 You'll need to have these libraries with equivalent development versions to
 build the Anura engine:
 
- 1.50.0 <= boost_iostreams     <= 1.73.0
- 1.50.0 <= boost_filesystem    <= 1.73.0
- 1.50.0 <= boost_regex         <= 1.73.0
- 1.50.0 <= boost_asio          <= 1.73.0
- 1.50.0 <= boost_system        <= 1.73.0
- 1.50.0 <= libboost-locale-dev <= 1.73.0
+ 1.50.0 <= boost_iostreams     <= 1.74.0
```

Comment:
Not sure if these borders are true and not immediately sure how to conveniently find out either.

Reply:
I am willing to accept it at face value if it doesn't cause problems. 1.73 is fairly old at this point, I think.